Muvr/DigestAlgotithm.swift

```diff
@@ -0,0 +1,68 @@
+import Foundation
+
+typealias HmacDigest = [Int8]
+
+enum DigestAlgorithm {
+    case MD5, SHA1, SHA224, SHA256, SHA384, SHA512
```

Comment:
If this is a copy-and-paste from somewhere, can we just use the digest we require in the AWS code?